Jenkinsfile

```diff
@@ -779,6 +748,66 @@ stage('Test') {
       Utils.markStageSkippedForConditional('python3: i386')
     }
   },
+  'test: Hexagon': {
+    if (!skip_ci && is_docs_only_build != 1) {
+      node('CPU') {
+        ws("workspace/exec_${env.EXECUTOR_NUMBER}/tvm/test-hexagon") {
+          timeout(time: max_time, unit: 'MINUTES') {
+            try {
+              init_git()
+              unpack_lib('hexagon', tvm_multilib)
+              ci_setup(ci_hexagon)
+              cpp_unittest(ci_hexagon)
+              sh (
+                script: "${docker_run} ${ci_hexagon} ./tests/scripts/task_build_hexagon_api.sh",
+                label: 'Build Hexagon API',
+              )
+              sh (
+                script: "${docker_run} ${ci_hexagon} ./tests/scripts/task_python_hexagon.sh",
+                label: 'Run Hexagon tests',
+              )
+              sh (
+                script: "${docker_run} ${ci_hexagon} ./tests/scripts/task_python_hexagon_simulator.sh",
```

Comment:
I'm not sure if it's doable, but this step can run in parallel with the step at line 766.

Reply:
We could especially now that we're installing GNU `parallel` onto the Docker images, but only the longest step really matters so we don't have to mess with this unless we get the CPU/GPU tests under 30 minutes or so